elastic/datadog_checks/elastic/data/conf.yaml.example

```diff
@@ -1,75 +1,116 @@
 init_config:
 
 instances:
-  # The URL where elasticsearch accepts HTTP requests. This will be used to
-  # fetch statistics from the nodes and information about the cluster health.
+
+## @param url - string - required
+## The URL where elasticsearch accepts HTTP requests. This is used to
+## fetch statistics from the nodes and information about the cluster health.
+# 
```

Comment:
nit: indentation doesn't look right here

Reply:
Yup, idk if I should indent the comment here since the `url` parameter starts the instance definition and so the `-` makes that the comments should not be indented.

Do you think we should ident it anyway ?

Reply:
I don't know, you tell me. It's indented here : https://github.com/DataDog/integrations-core/blob/master/apache/datadog_checks/apache/data/conf.yaml.example#L5-L8 🙂